presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/HiveMetastoreClient.java

```diff
@@ -112,4 +112,11 @@ boolean grantPrivileges(PrivilegeBag privilegeBag)
 
     boolean revokePrivileges(PrivilegeBag privilegeBag)
             throws TException;
+
+    /**
+     * send UGI(UserGroupInformation , abbreviated in hadoop) to Hive Metastore,
+     * tell Hive Metastore who am i
+     */
+    List<String> setUGI(String userName, List<String> groupNames)
```

Comment:
We can remove the `groupNames` parameter since it's always empty. Also, have the method return void since the result is not used.

Reply:
ok,I will fix it